lib/pure/uri.nim

```diff
@@ -302,12 +349,21 @@ proc combine*(base: Uri, reference: Uri): Uri =
 
 proc combine*(uris: varargs[Uri]): Uri =
   ## Combines multiple URIs together.
+  runnableExamples:
+    let foo = combine(parseUri("https://nim-lang.org/blog.html"), parseUri("/install.html"))
+    doAssert foo.hostname == "nim-lang.org"
+    doAssert foo.path == "/install.html"
   result = uris[0]
   for i in 1 ..< uris.len:
     result = combine(result, uris[i])
 
 proc isAbsolute*(uri: Uri): bool =
-  ## returns true if URI is absolute, false otherwise
+  ## Returns true if URI is absolute, false otherwise
```

Comment:
The most minor nitpick ever: put dot at the end :)

Reply:
a4dfee3094a055f3e8152aa0b98f12ff0f60c770